include/highfive/bits/H5Dataspace_misc.hpp

```diff
@@ -129,38 +129,9 @@ inline std::vector<size_t> DataSpace::getMaxDimensions() const {
     return details::to_vector_size_t(maxdims);
 }
 
-template <typename ScalarValue>
-inline DataSpace DataSpace::From(const ScalarValue& scalar) {
-    (void)scalar;
-    static_assert(
-        (std::is_arithmetic<ScalarValue>::value ||
-         std::is_enum<ScalarValue>::value ||
-         std::is_same<std::string, ScalarValue>::value),
-        "Only the following types are supported by DataSpace::From: \n"
```

Comment:
@alkino I thought a bit and we should keep this for users. For ourselves maybe we can disable this, but many users of HighFive don’t know templating and will be lost

Reply:
Do you think there is a way, not too complicated to show the message in case `details::inspector<T>` doesnt support T ?

Reply:
My concern is that it is hard to tell which types are accepted for `T` (for containers it is easy).

Reply:
And when it is compiled without Eigen (but you don't think about it) and the message is "Eigen::matrix is accepted".

Reply:
ok